src/utils/filterPlayers.js

```diff
@@ -0,0 +1,13 @@
+const filterPlayers = (players, filter) => {
+  if (!filter) {
+    return players;
+  }
+
+  if (filter.limit) {
+    return players.filter(player => player[filter.key] <= filter.limit);
```

Comment:
Didnt convert price like in `playerData` resolver, but can if it makes it more consistent